molgenis-questionnaires/src/main/frontend/src/store/getters.js

```diff
@@ -0,0 +1,42 @@
+const getAllFields = (chapter, data) => {
+  return chapter.children.reduce((accumulator, child) => {
```

Comment:
what if chapter has no children ? , adding flow typing would fix this

Reply:
Then you have an empty array, and all the components are designed to work with an empty array as "base" state. But flow typing should be added yes